feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -5,13 +5,14 @@
       #searchTerm
       type="text"
       formControlName="searchTerm"
+      [attr.aria-label]="'asm.customerSearch.searchTerm.label' | cxTranslate"
       placeholder="{{ 'asm.customerSearch.searchTerm.label' | cxTranslate }}"
     />
     <cx-form-errors
       [control]="customerSelectionForm.get('searchTerm')"
     ></cx-form-errors>
   </label>
-  <button type="submit">
+  <button type="submit" [class.active]="selectedCustomer">
```

Comment:
I suspect this is a breaking change.
Also, I think the correct way to implement this is to enable/disable the submit button based on the form validity.  This way the validation rules are driven only by the form in the component and all the template needs to know is if the form is valid.

Reply:
I discussed this with others. While we do not yet have a clearly defined rule on how to handle these cases, we agree that we can't guarantee that adding classes won't collide with customer classes/styling.  To unblock things in this PR and to be able to move forward, let's use the feature level directive in the html to add the class in 5.1+.